### ISSUE TITLE
Tiny ImageNet Eval Top1 100%?

### ISSUE THREAD

**Issue by PaulTHong** [Beckschen/TransMix#9]
Hey, I have a question to bother you. I use this code to train Tiny ImageNet, all the configs are reserved. But after dozens of epochs, the top1 accuracy on eval is 100%. Is there anything wrong? I can't find the problem. And I observe that the test acc is higher than the test EMA acc in former epochs, then why need I to use EMA? Thanks.

**Comment by kevin-ssy** [comment]
I guess there might be something wrong with your train_dataset and val_dataset. Please check your path of the validation dataset carefully to make sure that it is not the same as your training dataset.

**Comment by PaulTHong** [comment]
Yeah，I have checked before, but I have not found any problem. So I am very confused.

**Comment by PaulTHong** [comment]
I just follow your configs and instruction in readme. Running ImageNet 2012 is OK,  the layout of image directory etc. of Tiny are all the same as ImageNet.

**Comment by kevin-ssy** [comment]
Maybe there's something wrong with your dataset? I don't think this bug can be caused by our codebase.

**Comment by PaulTHong** [comment]
But when I run other network with the same dataset, its also OK. Yeah, there is a high probability that something is wrong. I have tried to figure out while I didn't success. Have you experimented on other datasets than ImageNet 2012?

**Comment by kevin-ssy** [comment]
Sorry for this codebase we only tried on ImageNet 1k. But it is highly similar to the timm training script. I think it should be compatible with other datasets since timm has been widely validated.
